sql/updates/latest-dev.sql

```diff
@@ -397,3 +397,6 @@ CREATE FUNCTION _timescaledb_functions.constraint_clone(constraint_oid OID,targe
 DROP FUNCTION IF EXISTS _timescaledb_functions.chunks_in;
 DROP FUNCTION IF EXISTS _timescaledb_internal.chunks_in;
 
+
+CREATE FUNCTION _timescaledb_functions.ts_replace_now_mock()
```

Comment:
This should probably be "ts_now_mock()"? The current "ts_replace_now_mock()" sounds like the function that does the replacement, I was staring at it trying to understand why you need it in SQL :smiley:

Reply:
Also probably we don't need it in the production schema, but not sure where to put it.

Reply:
We have a file `debug_build_utils.sql` that is used for utilities that should only be available in debug builds.

Not sure you need to have this function in the upgrade/downgrade scripts since it is only going to be used for regression test runs in debug builds.